src/main/java/org/sonarqube/gradle/SonarPropertyComputer.java

```diff
@@ -217,7 +219,16 @@ private static void computeScanAllProperties(Project project, Map<String, Object
       .map(File::toPath)
       .collect(Collectors.toSet());
 
-    SourceCollector visitor = new SourceCollector(allModulesExistingSources, skippedDirs, Set.of(), false);
+    Path root = Path.of(findProjectBaseDir(properties));
+    Set<Path> excludedFiles = extractReportPaths(properties).stream()
```

Comment:
We could do it but it makes testing the logic a little more complicated: we need to have a `sonar.projectBaseDir` defined in all tests otherwise the code crashes because it does not seem to be built to work around the missing variable.
I will see what I can do

Reply:
If testing doesn't turn out to be too painful I would prefer having the logic in the SonarUtils, for readability and consistency